### DISCUSSION TITLE
Mutation Events Deprecated

### DISCUSSION THREAD

**Discussion by ngalioth** [the1812/Bilibili-Evolved#4513] [功能建议 / Ideas]
Chromium has officially deprecated mutation events, and has a plan to remove support starting with version 127, which goes to stable release on [July 30, 2024](https://chromiumdash.appspot.com/schedule).
Right now if we manually disable this api in flags [Enable (deprecated) synchronous mutation events], the plugin won't load. Maybe we should deprecate it as well

PS:
the problems is on tampermonkey, not the plugin.  In fact, all scripts fail to load with the api above disabled. Sorry for not double checking again.

**Comment by the1812** [comment]
We never use Mutation Events.
